docs/advanced-topics/metrics.rst

```diff
@@ -0,0 +1,154 @@
+Workflow Metrics
+================
+
+.. note:: Dockstore Workflow Metrics is currently in preview mode. Also note that the screenshots below were taken on our staging site.
+
+What are workflow metrics?
+------------------------------------
+Workflow metrics are metrics of workflow executions on a platform. Metrics include things like execution status and the resources used during the execution. 
```

Comment:
On Charles' note, maybe make this sound a little more accessible to the end user 
i.e. 
execution status -> how often workflows succeeded or failed